.travis/runtox

```diff
@@ -16,23 +16,34 @@ ME=$(basename $0)
 SCRIPTDIR=$(readlink -f $(dirname $0))
 BANNERSIZE=90
 
-# LSR_MOLECULE_DOCKER_VERSION is the version string to pass
+# LSR_MOLECULE_DRIVER is the name of all of the following:
+# * the command line program
+# * the value of molecule driver.name
+# * the pypi package to install via pip
+# LSR_MOLECULE_DRIVER_VERSION is the version string to pass
 # to pip install - this is usually a comparison operator
 # followed by a version number e.g. '<4.3' but in reality
 # may be anything that pip allows to follow a package name
 # in an install specification
-if [ -z "${LSR_MOLECULE_DOCKER_VERSION:-}" ]; then
+if [ -z "${LSR_MOLECULE_DRIVER:-}" ]; then
   if type docker > /dev/null 2>&1; then
-    DOCKER_SERVER_API_VERSION=$(docker version --format '{{.Server.APIVersion}}')
-    case "$DOCKER_SERVER_API_VERSION" in
-    1.3[0-8]|1.[0-2]*) LSR_MOLECULE_DOCKER_VERSION='<4.3';;
-    *) LSR_MOLECULE_DOCKER_VERSION="";;
-    esac
+    LSR_MOLECULE_DRIVER=docker
+  elif type podman > /dev/null 2>&1; then
```

Comment:
It might be a better idea to first check for podman's existence and then for docker. On recent Fedora, the docker command is an alias for podman and I believe there is no normal docker because it does not work with cgroups v2.